cypress/integration/e2e/atom.interactivity.spec.js

```diff
@@ -61,4 +64,37 @@ describe('Atom Interactivity', function () {
         });
         */
     });
+
+    describe('Guide Atom', function () {
+        it('should render', function () {
+            cy.visit(`/Article?url=${guideUrl}`);
+            cy.get('[data-snippet-type="guide"]').should('be.visible');
+        });
+
+        it('should expand on click', function () {
+            cy.visit(`/Article?url=${guideUrl}`);
+            cy.get('[data-snippet-type="guide"]').should('be.visible');
+            cy.get('[data-snippet-type="guide"]')
+                .contains('Show')
+                .should('not.be.null');
+            cy.get('[data-snippet-type="guide"]').click();
+            cy.get('[data-snippet-type="guide"]')
+                .contains('Hide')
+                .should('not.be.null');
+        });
+
+        it('should show feedback message when like is clicked', function () {
+            cy.visit(`/Article?url=${guideUrl}`);
+            cy.get('[data-snippet-type="guide"]').click();
+            cy.get('[data-testid="like"]').click();
+            cy.get('[data-testid="feedback"]').should('be.visible');
+        });
+
+        it('should show feedback message when dislike is clicked', function () {
+            cy.visit(`/Article?url=${guideUrl}`);
+            cy.get('[data-snippet-type="guide"]').click();
+            cy.get('[data-testid="dislike"]').click();
+            cy.get('[data-testid="feedback"]').should('be.visible');
+        });
```

Comment:
I think as this is generally copied, you could abstract into a function and pass in the `type` so you'd have a function:

```
const atomTests = (type, url) = {
... all your tests
it('should show feedback message when dislike is clicked', function () {
            cy.visit(`/Article?url=${url}`);
            cy.get(`[data-snippet-type="${type}"]`).click(); // I think this will work!?
            cy.get('[data-testid="dislike"]').click();
            cy.get('[data-testid="feedback"]').should('be.visible');
        });
...
}
```

and call them like 

```
atomTests('qanda', qandaUrl);
atomTests('guide', guideUrl);
```

That way, adding and amending these similar types should be quick (and you can do the other atom type easily)